tests/testsuite/kubevirtresource.go

```diff
@@ -117,6 +117,7 @@ func AdjustKubeVirtResource() {
 		featuregate.KubevirtSeccompProfile,
 		featuregate.ObjectGraph,
 		featuregate.DeclarativeHotplugVolumesGate,
+		featuregate.PasstIPStackMigration,
```

Comment:
I think this change requires a dedicated commit with proper reasoning.

Reply:
There is no need to enable this FG for the entire test suite.
Please enable it only for the tests using the passt binding plugin:
- tests/network/bindingplugin_passt.go
- tests/network/vmi_istio.go

Reply:
There is also a passt test in `tests/network/bindingplugin.go`.